DependencyInjection/NelmioSecurityExtension.php

```diff
@@ -136,4 +147,51 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setParameter('nelmio_security.forced_ssl.hosts', $config['forced_ssl']['hosts']);
         }
     }
+
+    private function buildDirectiveSetDefinition($config, $type)
```

Comment:
For the record @kerneldemon, why did you updated this file, write all this code? Whas it buggy?